lib/rubocop/cop/style/unneeded_condition.rb

```diff
@@ -44,6 +44,8 @@ def autocorrect(node)
           lambda do |corrector|
             if node.ternary?
               corrector.replace(range_of_offense(node), '||')
+            elsif node.modifier_form?
+              corrector.replace(node.source_range, "#{node.if_branch.source} || nil")
```

Comment:
This change looks good, but this line is being warned by `Metrics/LineLength` cop.
https://travis-ci.org/rubocop-hq/rubocop/jobs/390954328#L599-L601

Reply:
(same for `a || a`, `b && b`, etc)

Reply:
> (same for a || a, b && b, etc)

Can you provide more information? I don't know what this comment means in this context.